.gitlab/deps_build/deps_build.yml

```diff
@@ -94,3 +94,36 @@ build_processed_btfhub_archive:
     - inv -e system-probe.process-btfhub-archive --branch $BTFHUB_ARCHIVE_BRANCH
     - $S3_CP_CMD btfs-x86_64.tar $S3_DD_AGENT_OMNIBUS_BTFS_URI/$BTFHUB_ARCHIVE_BRANCH/btfs-x86_64.tar --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
     - $S3_CP_CMD btfs-arm64.tar $S3_DD_AGENT_OMNIBUS_BTFS_URI/$BTFHUB_ARCHIVE_BRANCH/btfs-arm64.tar --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
+
+.build_libpcap_common:
+  rules:
+    - !reference [.except_mergequeue]
+    - when: on_success
+  stage: deps_build
+  variables:
+    KUBERNETES_CPU_REQUEST: 2
+  script:
+    - inv -e system-probe.build-libpcap --version=$LIBPCAP_VERSION --sha256sum=$LIBPCAP_SHA256SUM
+    - strings $CI_PROJECT_DIR/dev/lib/libpcap.a | grep -E "^libpcap version $LIBPCAP_VERSION" || exit 1 # ensure we're able to match the expected version
+  timeout: 10m
+  cache:
+    policy: pull-push
+    key: libpcap-$LIBPCAP_VERSION-$ARCH
```

Comment:
💭 I'm thinking whether we'd win anything by adding the sha to the cache key (something I may instinctively do).